packages/web/src/consts.ts

```diff
@@ -0,0 +1 @@
+export const BROWSER_EVENT_DOMAIN = 'browser';
```

Comment:
The naming convention for consts like those is `defaultOptionValue`.

Reply:
I was thinking of this as a otel rum attribute convention, not a kowalski option specifically

Reply:
that is, it has uses outside of configuring the agent